CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Master (Unreleased)
 
+## 1.21.1 (2017-12-14)
```

Comment:
This line must go. The "new version header" is added only when a new release is cut, not in individual feature PRs :-)